src/s1reader/s1_burst_slc.py

```diff
@@ -468,59 +492,66 @@ def geometrical_and_steering_doppler(self, xstep=500, ystep=50):
         with azimuth time.
         Parameters
         ----------
-        xstep: int
-            Spacing along x direction [pixels]
-        ystep: int
-            Spacing along y direction [pixels]
+        range_step: int
+            Spacing along x/range direction [meters]
+        az_step: int
+            Spacing along y/azimuth direction [seconds]
 
         Returns
         -------
-        x : int
-           The index of samples in range direction as an 1D array
-        y : int
-           The index of samples in azimuth direction as an 1D array
-        total_doppler : float
-           Total Doppler which is the sum of the geometrical Doppler and
-           beam steering induced Doppler [Hz] as a 2D array
+           LUT2D object of total doppler in Hz as a function of the azimuth
+           time and slant range, or range and azimuth indices.
+           This correction needs to be added to the SLC tagged azimuth time to
+           get the corrected azimuth times.
         """
+        range_arange, az_arange = self._steps_to_aranges(range_step, az_step)
```

Comment:
what do you think of the names `range_arr/az_arr` or `range_vec/az_vec`  to indicate they're arrays, rather than `_arange` (which says what function created them)?
Not a hard ask, just thinking that `range_arange` sounds a little funny haha

Reply:
Agreed `_arange` is awkward. Will switch to `_vec`.